packages/features/bookings/lib/handleNewBooking/validateBookingTimeIsNotOutOfBounds.ts

```diff
@@ -52,11 +58,13 @@ export const validateBookingTimeIsNotOutOfBounds = async <T extends ValidateBook
   if (timeOutOfBounds) {
     const error = {
       errorCode: "BookingTimeOutOfBounds",
-      message: `EventType '${eventType.eventName}' cannot be booked at this time.`,
+      message: `EventType '${eventType.title}' cannot be booked at this time. Reason: ${cause}`,
     };
+
     logger.warn({
-      message: `NewBooking: EventType '${eventType.eventName}' cannot be booked at this time.`,
+      message: `NewBooking: EventType '${eventType.title}' cannot be booked at this time. Reason: ${cause}`,
```

Comment:
We also add cause here to have a clearer picture of what the cause of this failure is.